core/src/main/java/cocktailofferer/service/image/command/CreateImageMetadataCommandResult.java

```diff
@@ -0,0 +1,11 @@
+package cocktailofferer.service.image.command;
+
+import lombok.Builder;
+
+@Builder
+public record CreateImageMetadataCommandResult(
```

Comment:
On our last project we decided to simplify this design. Look at already existing Cocktail services. Command services don't have to return data.

Reply:
Why? For example, I want to get id of created image

Reply:
You will have to return the ID. Nothing more.

Reply:
But, if I want file id too (new attribute in our application)

Reply:
You get all data from query services.